build.js

```diff
@@ -20,11 +19,16 @@ const nodeMajorVersion = Number(process.version.match(/(?<=^v)\d+/));
 async function rmDistDir() {
   try {
     await stat(outdir);
-    if (nodeMajorVersion >= 12) {
-      await rmdir(outdir, { recursive: true });
+    if (nodeMajorVersion >= 14) {
+      await rm(outdir, { recursive: true });
```

Comment:
How can this work if the variable is defined below?

Reply:
It doesn't. I think it's time to take a break 😅